pontoon/contributors/static/js/contributor.js

```diff
@@ -1,21 +1,26 @@
 $(function() {
 
-  var input = $('#username input');
+  var username = $('#profile-username');
+  var email = $('#profile-email');
 
-  // Save user name handler
+  // Save user profile handler
   function save() {
     $.ajax({
-      url: '/save-user-name/',
+      url: '/save-user-profile/',
       type: 'POST',
       data: {
         csrfmiddlewaretoken: $('#server').data('csrf'),
-        first_name: $.trim(input.val())
+        first_name: $.trim(username.val()),
+        email: $.trim(email.val())
       },
       success: function(data) {
         if (data === "ok") {
-          input.blur();
           Pontoon.endLoader('Thank you!');
```

Comment:
Do you mind changing this by ``Changes saved!`` instead of this very confusing ``Thank you`` while you're working on this? It's bugging me... :)